### PR TITLE
[OSDOCS#7176]:Added documentation for AWS custom tags.

### PR DESCRIPTION
**Pull request by AedinC**
Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-7176

Link to docs preview:

- [Creating a cluster with customizations using the CLI](https://66140--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations#rosa-sts-creating-cluster-customizations-cli_rosa-sts-creating-a-cluster-with-customizations)
- [Interactive cluster creation mode options](https://66140--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-interactive-mode-reference#rosa-sts-interactive-cluster-creation-mode-options_rosa-sts-interactive-mode-reference)
- [Managing objects with the ROSA CLI](https://66140--docspreview.netlify.app/openshift-rosa/latest/cli_reference/rosa_cli/rosa-manage-objects-cli#rosa-create-cluster-command_rosa-managing-objects-cli)

Peer review:
- [x] Peer reviewer has approved this change.

SME review:
- [x] SME has approved this change.

QE review:
- [x] QE has approved this change.  

Additional information:

### CODE REVIEW

File: modules/rosa-create-objects.adoc [hunk 1]
```diff
@@ -202,6 +202,16 @@ When using `--private-link`, the `--subnet-ids` argument is required and only on
 |--support-role-arn string
 |The ARN of the role used by Red Hat Site Reliabilty Engineers (SREs) to enable access to the cluster account to provide support.
 
+|--tags
+a|Tags that are used on resources created by {product-title} in AWS. These are comma separated, for example: "key value, foo bar".
+[IMPORTANT]
+====
+{product-title} only supports custom tags to Red Hat OpenShift resources during cluster creation. Once added, the tags cannot be removed or edited.
+Tags that are added by Red Hat must not be removed to comply with Red Hat production service level agreements (SLAs).
```
Comment (ShaunaDiaz) [inline modules/rosa-create-objects.adoc:210]:
```suggestion
Tags that are added by Red Hat are required for clusters to stay in compliance with Red Hat production service level agreements (SLAs). These tags must not be removed.
```

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc [hunk 2]
```diff
@@ -235,24 +235,26 @@ I: Using arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Worker-Role for the
...
-   rosa create cluster --cluster-name <cluster_name> --role-arn arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Installer-Role --support-role-arn arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Support-Role --master-iam-role arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-ControlPlane-Role --worker-iam-role arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Worker-Role --operator-roles-prefix <cluster_name>-<random_string> --region us-east-1 --version 4.8.9 --compute-nodes 2 --machine-cidr 10.0.0.0/16 --service-cidr 172.30.0.0/16 --pod-cidr 10.128.0.0/14 --host-prefix 23 <9>
+   rosa create cluster --cluster-name <cluster_name> --role-arn arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Installer-Role --support-role-arn arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Support-Role --master-iam-role arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-ControlPlane-Role --worker-iam-role arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Worker-Role --operator-roles-prefix <cluster_name>-<random_string> --region us-east-1 --version 4.8.9 --compute-nodes 2 --machine-cidr 10.0.0.0/16 --service-cidr 172.30.0.0/16 --pod-cidr 10.128.0.0/14 --host-prefix 23 <10>
```
Comment (ShaunaDiaz) [inline modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:257]: If it's possible to break this command up with backslashes, it should be done. Very long render.

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc [hunk 3]
```diff
@@ -273,8 +275,17 @@ The Instance Metadata Service settings cannot be changed after your cluster is c
...
 If you specified custom ARN paths when you created the associated account-wide roles, the custom path is automatically detected. The custom path is applied to the cluster-specific Operator roles when you create them in a later step.
 ====
-<6> Optional: Multiple availability zones are recommended for production workloads. The default is a single availability zone.
-<7> Optional: Enable this option if you are using your own AWS KMS key to encrypt the control plane, infrastructure, worker node root volumes, and PVs. Specify the ARN for the KMS key that you added to the account-wide role ARN in the preceding step.
+<6> Optional: Specify a tag that is used on all resources created by {product-title} in AWS. Tags are comma separated, for example: "key value, foo bar".
++
+[IMPORTANT]
+====
+{product-title} only supports custom tags to Red Hat OpenShift resources during cluster creation. Once added, the tags cannot be removed or edited.
+Tags that are added by Red Hat must not be removed to comply with Red Hat production service level agreements (SLAs).
```
Comment (ShaunaDiaz) [inline modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:283]:
```suggestion
Tags that are added by Red Hat are required for clusters to stay in compliance with Red Hat production service level agreements (SLAs). These tags must not be removed.
```

File: modules/rosa-sts-interactive-cluster-creation-mode-options.adoc [hunk 4]
```diff
@@ -51,7 +51,14 @@ The ROSA with Hosted Control Planes functionality is currently offered as a Tech
 |Specify if you want to use a pre-configured OIDC configuration or if you want to create a new OIDC configuration as part of the cluster creation process.
 
 |`Tags (optional)`
-|Specify a tag that is used on all resources created by ROSA in AWS. Tags are comma separated, for example: "key value, foo bar".
+|Specify a tag that is used on all resources created by {product-title} in AWS. Tags are comma separated, for example: "key value, foo bar".
+[IMPORTANT]
+====
+{product-title} only supports custom tags to Red Hat OpenShift resources during cluster creation. Once added, the tags cannot be removed or edited.
+Tags that are added by Red Hat must not be removed to comply with Red Hat production service level agreements (SLAs).
```
Comment (ShaunaDiaz) [inline modules/rosa-sts-interactive-cluster-creation-mode-options.adoc:58]:
```suggestion
Tags that are added by Red Hat are required for clusters to stay in compliance with Red Hat production service level agreements (SLAs). These tags must not be removed.
```

**Review by ShaunaDiaz — COMMENTED** [review]
Looks good!

**Comment by arendej** [comment]
In the text block..

> Red Hat OpenShift Service on AWS does not support adding additional tags outside of Red Hat OpenShift to Red Hat OpenShift resources. These tags can be lost when AWS resources are deleted or recreated by the Red Hat OpenShift operators. In these cases, you might need custom solutions or tools to reconcile the tags and keep them intact.

I suggest changing this to

> Red Hat OpenShift Service on AWS does not support adding additional tags outside of ROSA cluster managed resources. These tags can be lost when AWS resources are managed by the ROSA cluster. In these cases, you might need custom solutions or tools to reconcile the tags and keep them intact.

**Comment by AedinC** [comment]
> In the text block..
> 
> > Red Hat OpenShift Service on AWS does not support adding additional tags outside of Red Hat OpenShift to Red Hat OpenShift resources. These tags can be lost when AWS resources are deleted or recreated by the Red Hat OpenShift operators. In these cases, you might need custom solutions or tools to reconcile the tags and keep them intact.
> 
> I suggest changing this to
> 
> > Red Hat OpenShift Service on AWS does not support adding additional tags outside of ROSA cluster managed resources. These tags can be lost when AWS resources are managed by the ROSA cluster. In these cases, you might need custom solutions or tools to reconcile the tags and keep them intact.

Thanks @arendej. Have made those changes. Can you please give it a LGTM if you are happy with it?

File: modules/rosa-sts-interactive-cluster-creation-mode-options.adoc [hunk 5]
```diff
@@ -51,7 +51,14 @@ The ROSA with Hosted Control Planes functionality is currently offered as a Tech
 |Specify if you want to use a pre-configured OIDC configuration or if you want to create a new OIDC configuration as part of the cluster creation process.
 
 |`Tags (optional)`
-|Specify a tag that is used on all resources created by ROSA in AWS. Tags are comma separated, for example: "key value, foo bar".
+|Specify a tag that is used on all resources created by {product-title} in AWS. Tags are comma separated, for example: "key value, foo bar".
```
Comment (bchandra-ocp) [inline modules/rosa-sts-interactive-cluster-creation-mode-options.adoc:54]: To imply that this is same as tags, suggest to add a statement that 'Tags can help you manage, identify, organize, search for, and filter resources within AWS'.

Comment (AedinC) [reply modules/rosa-sts-interactive-cluster-creation-mode-options.adoc]: > To imply that this is same as tags, suggest to add a statement that 'Tags can help you manage, identify, organize, search for, and filter resources within AWS'.

Thanks @bchandra-ocp. Have made that change.

Comment (AedinC) [reply modules/rosa-sts-interactive-cluster-creation-mode-options.adoc]: @bchandra-ocp Can you please give this a LGTM if you are happy with it? Thanks!

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM!

**Pull request merged**